File: wrappers/java/src/main/java/org/hyperledger/indy/sdk/DuplicateWalletTypeException.java
package org.hyperledger.indy.sdk;

public class DuplicateWalletTypeException extends IndyException
{
	private final static String message = "A wallet type with the specified name has already been registered.";

	DuplicateWalletTypeException(int sdkErrorCode) 
    {
    	super(message, sdkErrorCode);
    }
}

File: wrappers/dotnet/indy-sdk-dotnet/WalletApi/DuplicateWalletTypeException.cs
namespace Hyperledger.Indy.WalletApi
{
    /// <summary>
    /// Exception thrown when registering a wallet type that has already been registered.
    /// </summary>
    public class DuplicateWalletTypeException : IndyException
    {
        const string message = "A wallet type with the specified name has already been registered.";

        /// <summary>
        /// Initializes a new DuplicateWalletTypeException.
        /// </summary>
        internal DuplicateWalletTypeException() : base(message, (int)ErrorCode.WalletTypeAlreadyRegisteredError)
        {

        }
    }

}
